operators-manual/full-parameter-list.md

```diff
@@ -5818,7 +5818,7 @@ Time in seconds to spool up the motors from zero to min throttle.
 
 *Note: This parameter is for advanced users*
 
-This is a scaling factor for vehicles with a vertical booster motor used for extra lift. It is used with electric multicopters that have an internal combusion booster motor for longer endurance. The output to the BoostThrottle servo function is set to the current motor thottle times this scaling factor. A higher scaling factor will put more of the load on the booster motor. A value of 1 will set the BoostThrottle equal to the main throttle.
```

Comment:
This text is generated from metadata in the ardupilot repository, so it will come again unless it is fixed there.